test/algorithms/classifiers/test_vqc.py

```diff
@@ -12,21 +12,100 @@
 
 """ Test Neural Network Classifier """
 
+from __future__ import annotations
+
+from test import QiskitMachineLearningTestCase
 import functools
+import itertools
 import unittest
-from test import QiskitMachineLearningTestCase
-
-from typing import Callable
 
 import numpy as np
 import scipy
-from ddt import ddt, data
+from ddt import ddt, idata, data, unpack
+
+from sklearn.datasets import make_classification
+from sklearn.preprocessing import MinMaxScaler
+
 from qiskit import Aer
 from qiskit.algorithms.optimizers import COBYLA, L_BFGS_B
 from qiskit.circuit.library import RealAmplitudes, ZZFeatureMap
 from qiskit.utils import QuantumInstance, algorithm_globals
 
 from qiskit_machine_learning.algorithms import VQC
+from qiskit_machine_learning.exceptions import QiskitMachineLearningError
+
+RANDOM_SEED = 1111111
+algorithm_globals.random_seed = RANDOM_SEED
+
+# Set-up the quantum instances.
+SV_QUANTUM_INSTANCE = QuantumInstance(
+    Aer.get_backend("aer_simulator_statevector"),
+    seed_simulator=algorithm_globals.random_seed,
+    seed_transpiler=algorithm_globals.random_seed,
+)
+QASM_QUANTUM_INSTANCE = QuantumInstance(
+    Aer.get_backend("aer_simulator"),
+    shots=100,
+    seed_simulator=algorithm_globals.random_seed,
+    seed_transpiler=algorithm_globals.random_seed,
+)
+QUANTUM_INSTANCES_ = [SV_QUANTUM_INSTANCE, QASM_QUANTUM_INSTANCE]
```

Comment:
A silly question, why some variables end with "_" and some not? 
It looks weird here:
```python
        itertools.product(
            QUANTUM_INSTANCES_, NUM_QUBITS_LIST, FEATURE_MAPS, ANSATZES, OPTIMIZERS, DATASETS
        )
```

Reply:
This was just because I defined `COBYLA_` to avoid it clashing with the class import `COBYLA`. Then I renamed them all to match, then I thought that looked stupid and reverted the names. But, I missed that one.

I'm probably going to remove these globals to address Steve's comment below anyway.